docs/FunctionalTestBuildScripts.md

```diff
@@ -0,0 +1,13 @@
+# Build Scripts
```

Comment:
@VCOLAWTON, if you have a chance to take a quick look to see if there is anything that should be considered form a documentation standard perspective that'd be awesome.

Reply:
Hi Mary, looks good! I looked at other GH content and think that adding a sentence or two would make the section look more like information and less like loose leaf data.

Reply:
This document is definitely going to need some expansion, but I don't believe all of that is necessary at the moment. What we want to immediately focus on is, usability. Can someone come in, take a look at this document, and walk away knowing how to recreate your files? No. What will it take to get someone up and running?

Are these both pretty much basic hello world binaries? In which case, expand the paragraph beginning at line 3, to explain that the base scenario is going to be a simple hello world program.

As for where I see this document going, I think this would be a good location to begin describing our functional and baseline testing approach for binskim. But, again, that's going to be a longer process/discussion and should be a collaborative effort. I wouldn't include all of that as part of this PR.

Reply:
expand the paragraph beginning at line 3, to explain that the base scenario is going to be a simple hello world program. ---- done, thanks.

Reply:
This document is definitely going to need expansion, and it is absolutely necessary for usability. Not sure, what you mean by "all of that" but anything we add to this document will be tremendously helpful. I know people in your positions do not typically like to do this type of work and would rather be as simple as possible, so just let me know how you want me to help. Probably a good idea to loop Michael in, too!